nestjs/src/cross-check/utils/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const DEV_HOST = 'http://localhost:3000';
+
+export const PROD_HOST = 'https://app.rs.school';
```

Comment:
I guess this constants are not related to cross-check. Let's put them to `config.service`. Also by default we should be using localhost, if respective env variable is not set